api/can_predictions.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional
+import pydantic
+import datetime
+
+"""
+Documented https://www.dropbox.com/scl/fi/o4bec2kz8dkcxdtabtqda/CAN-V1-API-Draft.paper?dl=0&rlkey=f7elx3zmo6tt5s7mj5nvap9a0
+
+"""
+
+class _HospitalBeds(pydantic.BaseModel):
+    peakShortfall: int = pydantic.Field(default=0, description="Shortfall of beds needed at the peek hospitalizaitons")
+    peakDate: Optional[datetime.datetime] = pydantic.Field(default='', description="Date of peak hospitalizations")
```

Comment:
does json have a `null` field? probably should use `None` instead here

Reply:
removed that